WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -333,6 +342,18 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
     }
 
 
+    // MARK: - UIScrollViewDelegate
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        guard !didShowTooltip else {
+            return
+        }
+
+        if isDescriptionAICellVisible() {
+            tooltipPresenter?.showTooltip()
+            didShowTooltip = true
+        }
+    }
+
```

Comment:
❓ Is this the best place to display the tooltip? Why don't we use `viewDidAppear` or something similar?